docs/Invoke-ADTCommandWithRetries.mdx

```diff
@@ -28,7 +28,21 @@ This function invokes the specified cmdlet/function, accepting all of its parame
 Invoke-ADTCommandWithRetries -Command Invoke-WebRequest -Uri https://aka.ms/getwinget -OutFile "$($adtSession.DirSupportFiles)\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle"
 ```
 
-Downloads the latest WinGet installer to the SupportFiles directory.
+Downloads the latest WinGet installer to the SupportFiles directory. If the command fails, it will retry 3 times with 5 seconds between each attempt.
+
+### EXAMPLE 2
+```powershell
```

Comment:
Were these examples added by hand, or by our build system? They appear hand-added. If so, can you please:
1. Run `.\actions_bootstrap.ps1` from the root of our repository
2. Run `Invoke-Build -File .\src\PSAppDeployToolkit.build.ps1` to build the module, generating help files appropriately.

Reply:
They were added by hand. I tried running the build process but ran into a bunch of errors. I'll try to get it to run again in the morning.